boards/arm/96b_neonkey/doc/96b_neonkey.rst

```diff
@@ -0,0 +1,194 @@
+.. _96b_neonkey:
+
+96Boards Neonkey
+################
+
+Overview
+********
+
+96Boards Neonkey board is based on the STMicroelectronics STM32F411CE
+Cortex M4 CPU. Zephyr applications use the 96b_neonkey configuration
+to run on these boards.
+
+.. figure:: img/96b-neonkey-front.jpg
+     :width: 500px
+     :align: center
+     :height: 330px
+     :alt: 96Boards Neonkey
+
+     96Boards Neonkey
+
+This board acts as a sensor hub platform for all 96Boards compliant
+family products. It can also be used as a standalone board.
+
+Hardware
+********
+
+96Boards Neonkey provides the following hardware components:
+
+- STM32F411CE in UFQFPN48 package
+- ARM |reg| 32-bit Cortex |reg|-M4 CPU with FPU
+- 100 MHz max CPU frequency
+- 1.8V work voltage
+- 512 KB Flash
+- 128 KB SRAM
+- On board sensors:
+
+  - Temperature/Humidity
```

Comment:
Can we have more details about the supported sensors?
I've not been able to find more information though any of the link.
So difficult to see if supported in Zephyr or not

Reply:
Sure, will add the part name of sensors

Reply:
Don't see the part names added yet?

Reply:
That's my bad. Somehow the commit got reverted! Added now.